app/models/case_request.rb

```diff
@@ -5,10 +5,10 @@ class CaseRequest
     :confirmation_code,
     :fees
 
-  def initialize(opts = {})
-    @case_reference = opts[:case_reference]
-    @confirmation_code = opts[:confirmation_code]
-    @fees = []
+  def initialize(opts)
```

Comment:
Might it be better to then just have these as keyword arguments with `nil` defaults? Feels like we're just emulating that behaviour – considering we have no optional arguments here.